lib/cfg-parser.c

```diff
@@ -164,6 +164,7 @@ static CfgLexerKeyword main_keywords[] =
   { "batch_timeout",      KW_BATCH_TIMEOUT },
 
   { "read_old_records",   KW_READ_OLD_RECORDS},
+  { "no_data_delay",      KW_NO_DATA_DELAY},
```

Comment:
Opt:
I tried to find a better option name (with not much success :smile:), what do you think of the options below?
`fetch-retry-delay`, `time-fetch-retry`, `fetch-nodata-delay`?

Reply:
fetch-nodata-delay sounds good